### PR TITLE
[Post] ES6 - async/await

### PR DESCRIPTION
**Pull request by naholyr**

### CODE REVIEW

File: content/fr/articles/js/es2015/async-await.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+fait pas partie des propositions acceptées cette année. La [spécification](https://tc39.github.io/ecmascript-asyncawait/) est encore au
+stade 3 (candidate) à l'heure de cet article. Mais il ne fait plus aucun doute
+qu'elle (ou une variante) fera partie de la spécification ES2016.
+
+# Fonctions asynchrones
+
+Pour les traitements asynchrones, vous connaissez déjà [les promesses](http://putaindecode.io/fr/articles/js/es2015/promises/), qui ont
+évidemment remplacé les callbacks dans votre code. Si vous n'avez pas déjà fait
+la bascule vers les promesses (ne serait-ce que pour la propagation d'erreur),
+[les générateurs](http://putaindecode.io/fr/articles/js/es2015/generators/) ont
```
Comment (MoOx) [inline content/fr/articles/js/es2015/async-await.md]: No need for `http://putaindecode.io` in links :)

**Comment by MoOx** [comment]
Very nice.
Just some details (I am thinking out loud):
- what about moving the post into js/es2016/ since it's not in es2015
- what about updating the title to "ES2016 (?): async/await" (posts are not set in stone) an we might review it if relevant
- obviously we can still add it in the calendar view

**Comment by MoOx** [comment]
If you are ok, I can handle those changes an publish it. It's hard to get some reviews this days...

**Comment by magsout** [comment]
Très bien l'article. Super boulot que tu as fais sur ce ~calendrier~ 👍

**Comment by naholyr** [comment]
Les changements me vont Maxime ;) pas de souci !

**Comment by MoOx** [comment]
Merging and updating as I said.

**Comment by MoOx** [comment]
Reminder: [skip ci] doesn't work for merge commit :(

**Pull request merged**